packages/spruce-cli/src/commands/test.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+	this: 'object',
+	default: 'ITS THE DEFAULT'
```

Comment:
IT IS or  IT'S - unless "THE DEFAULT" belongs to IT

Reply:
File removed